duckduckgo/Android
# Use counts in leaderboard

Task/Issue URL: https://app.asana.com/0/361428290920652/1127123170200348

**Description**:
Switches to counts in the tracker leaderboard

**Steps to test this PR**:

Test migration and new data:
1. Switch to develop branch
1. Update`PrivacyDashboardViewModel.kt` variable `LEADERBOARD_MIN_DOMAINS_EXCLUSIVE` to 3
1. Run the app, visit a few pages and make sure the tracker leaderboard is showing
1. Switch to this branch
1. Run the app and ensure the migration succeeds
1. Visit the privacy dash, the leaderboard should reset
1. Visit a few pages and make sure the tracker leaderboard shows again

Check updated database:
1. Download the app.db database from app storage
1. Check the network_leaderboard table and ensure network entries are now counts not strings
1. Check the sites_visited table and ensure the entries are now counts not strings

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
